lib/ecto/repo.ex

```diff
@@ -1055,6 +1055,15 @@ defmodule Ecto.Repo do
   @doc """
   Returns true if the current process is inside a transaction.
 
+  If you are using the Ecto.Adapters.SQL.Sandbox in tests, note that even
```

Comment:
I would include backticks to enable auto-linking:

- `Ecto.Adapters.SQL.Sandbox`
- `in_transaction?/1`
- `transaction/2`

Reply:
Good idea. Done.